infrastructure/cdn-in-a-box/traffic_ops/Dockerfile

```diff
@@ -35,12 +35,16 @@ RUN set -o nounset -o errexit && \
 	if [[ "${RHEL_VERSION%%.*}" -eq 7 ]]; then \
 		use_repo=''; \
 		enable_repo=''; \
+		# needed for llvm-toolset-7-clang, which is needed for postgresql13-devel-13.2-1PGDG, required by TO rpm
+		dnf -y install centos-release-scl-rh; \
```

Comment:
Because `centos-release-scl-rh` is installed, `devtoolset-7-gcc` ends up providing the `gcc` dependency instead of `gcc`, which ends up making `install_goose.sh` fail because `/opt/rh/devtoolset-7/root/usr/bin/gcc` is not in the PATH:

```
#11 [ 7/11] RUN ./install_goose.sh && rm ./install_goose.sh && dnf -y remove git && dnf clean all
#11 sha256:31d9b30b624f1d81c87461e7d564dc4d8d75ec98e04a912c21a9ca3b4a4848c5
#11 1.086 Now installing goose
#11 1.524 bitbucket.org/liamstask/goose (download)
#11 2.710 github.com/go-sql-driver/mysql (download)
#11 4.319 github.com/kylelemons/go-gypsy (download)
#11 5.474 github.com/lib/pq (download)
#11 6.978 github.com/mattn/go-sqlite3 (download)
#11 9.613 github.com/ziutek/mymysql (download)
#11 11.07 github.com/lib/pq/oid
#11 11.09 github.com/kylelemons/go-gypsy/yaml
#11 11.09 github.com/lib/pq/scram
#11 11.09 github.com/ziutek/mymysql/mysql
#11 11.10 github.com/go-sql-driver/mysql
#11 11.12 github.com/lib/pq
#11 11.13 github.com/mattn/go-sqlite3
#11 11.14 # github.com/mattn/go-sqlite3
#11 11.14 exec: "gcc": executable file not found in $PATH
#11 11.18 github.com/ziutek/mymysql/native
#11 11.33 github.com/ziutek/mymysql/godrv
#11 11.41 Error on line 23 of ./install_goose.sh
#11 ERROR: executor failed running [/bin/sh -c ./install_goose.sh && rm ./install_goose.sh && dnf -y remove git && dnf clean all]: exit code: 1
------
 > [ 7/11] RUN ./install_goose.sh && rm ./install_goose.sh && dnf -y remove git && dnf clean all:
------
executor failed running [/bin/sh -c ./install_goose.sh && rm ./install_goose.sh && dnf -y remove git && dnf clean all]: exit code: 1
```